apps/examples/testcase/le_tc/kernel/tc_termios.c

```diff
@@ -45,6 +45,18 @@ static void tc_termios_tcsetattr_tcgetattr(void)
 	}
 
 	sleep(1);
+	ret_chk = isatty(fileno(stdin));
+	TC_ASSERT_EQ("isatty", ret_chk, 0);
```

Comment:
when I run this code, it failed.
can you check this again?
originally, we want to test tcsetattr and tcgetattr with manual test.
(you can refer to this : http://suprem.sec.samsung.net/confluence/display/TINYARAC/Kernel+tcgetattr%2C+tcsetattr+Test+Guide)